ipaserver/plugins/domainlevel.py

```diff
@@ -48,6 +48,26 @@ def get_domainlevel_range(master_entry):
         return DomainLevelRange(0, 0)
 
 
+def check_conflict_entries(ldap, api):
+    """
+    Returns true if conflict entries exist in topology subtree
+    """
+
+    container_dn = DN(
+        ('cn', 'ipa'),
+        ('cn', 'etc'),
+        api.env.basedn
+    )
+
+    try:
+        ldap.get_entries(
+            filter="(&(nsds5replconflict=*)(|(objectclass=ldapsubentry)(objectclass=*)))",
+            base_dn=container_dn,
+            scope=ldap.SCOPE_SUBTREE)
+        return True
```

Comment:
Please raise the InvalidDomainLevel here directly, there is no point in returning a boolean and then checking it in the caller to raise exception.

This leads to cleaner code uncluttered with extraneous boolean checks.